test/test-resourcetiming-decompression.js

```diff
@@ -96,6 +96,26 @@
                 expect(ResourceTimingDecompression.getInitiatorTypeFromIndex(3)).to.be("script");
                 expect(ResourceTimingDecompression.getInitiatorTypeFromIndex(4)).to.be("css");
                 expect(ResourceTimingDecompression.getInitiatorTypeFromIndex(5)).to.be("xmlhttprequest");
+                // expect(ResourceTimingDecompression.getInitiatorTypeFromIndex(6)).to.be("html");
```

Comment:
Since html and navigation could both be mapped to '6', I'm not sure what's the preferred decompression.

Reply:
Let's go with `navigation` here since that's the spec'd one.